source/_components/homekit_controller.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Homekit controller support"
+description: "Instructions how to integrate your Homekit devices within Home Assistant."
+date: 2018-03-19 21:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: homekit.png
+ha_category: Hub
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+[Homekit](https://developer.apple.com/homekit/) controller integration for Home Assistant allows you to connect Homekit accessories to Home Assistant
```

Comment:
We should add some kind of note to explain how this component differs from the existing HomeKit component? So the purpose of both is clear to the users.

Reply:
Done

Reply:
I don't see it yet. Forgot to push?

Reply:
Oops! Really done.